docs/dev.md

```diff
@@ -89,6 +89,16 @@ Run tests:
 docker-compose exec app pytest --cov=src --cov=migrations/versions tests
 ```
 
+## Aggregate MDS
```

Comment:
i think it should be placed in here since it is more dev related

Reply:
could you add it to the table of contents